engines/bastion_katello/lib/bastion_katello/engine.rb

```diff
@@ -14,12 +14,6 @@ class Engine < ::Rails::Engine
 
       Bastion.register_plugin(
         :name => 'bastion_katello',
-        :javascript => proc do
-          [
-            javascript_include_tag(*webpack_asset_paths('katello:common', :extension => 'js'), "data-turbolinks-track" => true),
```

Comment:
This broke the "Sync Status" (`/katello/sync_management`) page, as the console now contains the following error:
```
63:2 Uncaught ReferenceError: localize is not defined
```

Reply:
Looking into that

Reply:
That's an ERB page, I wonder why we're even loading Katello JS here :thinking:

Reply:
created https://github.com/Katello/katello/pull/10873